utils/policycoreutils/Makefile

```diff
@@ -48,6 +48,12 @@ HOST_MAKE_FLAGS += \
 	ETCDIR=$(STAGING_DIR_HOSTPKG)/etc
 
 define Package/policycoreutils/install
+	# Remove programs provided by BusyBox.
+	$(RM) -f $(PKG_INSTALL_DIR)/sbin/restorecon
+	$(RM) -f $(PKG_INSTALL_DIR)/sbin/setfiles
+	$(RM) -f $(PKG_INSTALL_DIR)/usr/sbin/load_policy
+	$(RM) -f $(PKG_INSTALL_DIR)/usr/sbin/sestatus
+	$(RM) -f $(PKG_INSTALL_DIR)/usr/sbin/setsebool
```

Comment:
This won't work as those files are not present in `$(PKG_INSTALL_DIR)`
See https://github.com/openwrt/packages/blob/master/net/bridge-utils/Makefile for an example on how to replace busybox applets using `ALTERNATIVES`.

Reply:
The latest push adds the use of ALTERNATIVES.